src/MinFraud/Model/Subscores.php

```diff
@@ -109,11 +113,15 @@ class Subscores extends AbstractModel
 
     /**
      * @internal
+     *
+     * @deprecated
```

Comment:
When we have a suggested alternative or an explanation, I would add it here as I believe IDEs will show that message.